GitVersion.yml

```diff
@@ -0,0 +1,21 @@
+mode: Mainline

```

Comment:
Why are you going with the Mainline mode here?

According to the release process we are using the GitFlow mode, not mainline.

Currently the Release Process and the GitVersion configuration are in conflict, and assuming we do not want to automatically release a new version on every build then GitFlow would be the appropriate choice (or if no master branch, the GitHubFlow).

The config should probably be the same as we are using for the [Chocolatey CLI repository](https://github.com/chocolatey/choco/blob/bb2dc722fdcb2417fc2ddaf46e2645f80977f19c/GitVersion.yml).

Reply:
🤔 A number of the files I blatantly stole from the ChocoCCM repository 😂 . This was one such file. I've modified the file to be a close proximation to the Chocolatey CLI repository (need to ignore prior commits or the version number becomes 2022.10.25). Will see if this also fixes the PR GHA builds.

Reply:
The file needed to still have the ignore section so that it didn't pick up the `AU` release tags. I've got this updated and it seems to be working now 😄